libcob/common.h

```diff
@@ -2864,6 +2871,37 @@ COB_EXPIMP cob_field *cob_intr_bit_to_char		(cob_field *);
 COB_EXPIMP cob_field* cob_intr_hex_of (cob_field*);
 COB_EXPIMP cob_field* cob_intr_hex_to_char (cob_field*);
 
+
+
+/************************/
+/* Functions in jor.c */
+/************************/
+
+enum cob_jor_file_operation {
+/* order must match field_file_op_name[] in jor.c */
+	COB_JOR_WRITE_TRY = 0,
+	COB_JOR_WRITE_OK,
+	COB_JOR_READ_TRY,
+	COB_JOR_READ_OK,
+	COB_JOR_START_TRY,
+	COB_JOR_START_OK,
+	COB_JOR_OPEN_TRY,
+	COB_JOR_OPEN_OK,
+	COB_JOR_CLOSE_TRY,
+	COB_JOR_CLOSE_OK,
+	COB_JOR_AFTER_LAST_OPERATION
+};
```

Comment:
put that above, add a trailing "_MAX" entry and use that max entry as index instead of `COB_JOR_FILE_OPERATIONS `